src/features/calendar/components/CalendarDayView/Day/Event.tsx

```diff
@@ -40,91 +43,96 @@ const Event = ({ event }: { event: ZetkinEvent }) => {
   }
 
   return (
-    <NextLink href={getEventUrl(event)} passHref>
-      <Link color="inherit" underline="none">
-        <Box
-          display="flex"
-          flexDirection="row"
-          justifyContent="space-between"
-          padding={2}
-          sx={{ backgroundColor: 'white', borderRadius: '3px' }}
-          width="100%"
-        >
-          <Box alignItems="center" display="flex" gap={2.2}>
-            {/* Status */}
+    <Box sx={{ backgroundColor: 'white', borderRadius: '3px' }}>
```

Comment:
Yes, when I click the checkbox of the day view, it was not able to click the checkbox. It navigates to the event page instead because of `NextLink`. And `stopPropagagion` didn't work either. So I moved the Link instead

Reply:
It sounds weird to me that `stopPropagation()` would not work. Maybe `NextLink` handles events in the capture phase?